setup.py

```diff
@@ -22,7 +22,7 @@
 
 
 requirements = [
-    "strawberryfields>=0.15",
+    "strawberryfields>=0.19",

```

Comment:
Should we specify the minor version here too?
```suggestion
    "strawberryfields>=0.19.2",
```